packages/fx-core/resource/package.nls.json

```diff
@@ -719,13 +719,20 @@
   "core.SampleSelect.buttons.viewSamples": "View samples",
   "core.HostTypeOptionAzure.label": "Azure",
   "core.HostTypeOptionSPFx.label": "SharePoint Framework (SPFx)",
-  "core.updateBotIdsQuestion.title": "We will create a new bot for you to get started with Teams Toolkit",
-  "core.updateBotIdsQuestion.placeholder": "Unselect to keep with the original one.",
-  "core.updateContentUrlQuestion.title": "We will update the content url of following tab(s) for you to get started with Teams Toolkit",
-  "core.updateWebsiteUrlQuestion.title": "We will update the website url of following tab(s) for you to get started with Teams Toolkit",
-  "core.updateContentUrlOption.description": "update the content url from %s to %s",
-  "core.updateWebsiteUrlOption.description": "update the website url from %s to %s",
-  "core.updateUrlQuestion.placeholder": "Unselect to keep with the original url",
+  "core.updateBotIdsQuestion.title": "Create new bot(s) for debugging",
+  "core.updateBotIdsQuestion.placeholder": "Unselect to keep with the original value of botId",
+  "_core.updateBotIdsQuestion.placeholder.comment": "'botId' is the field name which should not be localized.",
+  "core.updateBotIdForBot.description": "Update botId %s to \"${{BOT_ID}}\" in manifest.json",
+  "_core.updateBotIdForBot.description.comment": "'botId' and '${{BOT_ID}}' should not be localized. 'manifest.json' is the file name which should also not be localized.",
```

Comment:
"which should also not be localized." -> "which should not be localized" to make it straightforward.